lisa/sut_orchestrator/azure/notifiers.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import json
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Type
+
+from lisa import messages, notifier, schema
+from lisa.environment import EnvironmentMessage, EnvironmentStatus
+from lisa.util import LisaException, constants
+
+from .. import AZURE
+from .common import AzureNodeSchema
+
+
+@dataclass
+class VmsizeBasicInfo:
+    name: str = ""
+    vm_family: str = ""
+    core_count: int = 0
+
+
+@dataclass
+class EnvironmentVmsizeInfo:
+    name: str
+    status: EnvironmentStatus = EnvironmentStatus.New
+    vm_sizes: List[VmsizeBasicInfo] = field(default_factory=list)
+
+
+class AzureNotifier(notifier.Notifier):
+    """
+    This notifier is used to collect the core usage information, which can then guide
+    the quota request in the subscription.
+    """
+
+    @classmethod
+    def type_name(cls) -> str:
+        return "azure_notifier"
```

Comment:
`azure_usage_counter`

Reply:
add `azure`, because it's azure specified. Maybe it needs other usage counters from other platforms.